udata/templates/dataset/display.html

```diff
@@ -238,18 +238,19 @@ <h3>{{ _('Informations') }}</h3>
                             {% endif %}
 
                             {# Source #}
+
                             {# ODS #}
                             {% if dataset.extras['ods:url'] %}
-                            <li>
-                                <a href v-tooltip title="{{ _('Remote source') }}"><span class="fa fa-fw fa-external-link "></span></a>
-                                <a href="{{ dataset.extras['ods:url'] }}">{{ _('Remote source') }}</a>
-                            </li>
-
+                                {% set source = dataset.extras['ods:url']  %}
```

Comment:
`{% set source = dataset.extras['ods:url'] or dataset.extras['remote_url'] %}` ? don't know if it works, maybe worth a try.